packages/components/src/Flex.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Box, BoxOwnProps, BoxProps } from './Box'
+
+export type FlexOwnProps = BoxOwnProps
+export type FlexProps = BoxProps
+
+// export declare const Flex: StyledComponent<FlexOwnProps, FlexProps>
+
+const __Flex = (props: FlexProps) => (
+  <Box
+    {...props}
+    sx={{
+      display: 'flex',
+      ...props.sx,
+    }}
+  />
+)
+
+__Flex.displayName = 'Flex'
+
+/**
+ * Use the Flex component to create flexbox layouts.
+ * @see https://theme-ui.com/components/flex
+ */
+export const Flex = __Flex as typeof __Flex & {
+  /**
+   * @deprecated
+   */
+  withComponent: (Component: React.ElementType) => React.ComponentType<BoxProps>
```

Comment:
Not clear why Flex needs this if Box implements it? Regardless, I'm happy with removing here as well